access/slackbot/main.go

```diff
@@ -225,6 +226,16 @@ func (a *App) Run(ctx context.Context) error {
 
 	ctx, cancel := context.WithCancel(ctx)
 	a.cancel = cancel
+	
+	clientCert, err := x509.ParseCertificate(a.tlsConf.Certificates[0].Certificate[0])
+	if err != nil {
+		cancel()
+		return trace.Wrap(err)
+	}
+
+	if time.Now().After(clientCert.NotAfter) {
+		log.Error("Teleport Auth certificates are expired.")
```

Comment:
I think it should be `log.Warning` since we don't terminate the app on such error.